### PR TITLE
update spark-defaults 

### PR DESCRIPTION
**Pull request by gshatz**
add the next properties to spark-defaults.conf:
spark.executor.cores=1
spark.cores.max=4
spark.executor.memory=2G

### CODE REVIEW

File: stable/jupyter/values.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 spark:
   hostname: spark-master
   port: 7077
-  executorCores: 2
+  executorCores: 1
+  coresMax: 4
```
Comment (bivas) [inline stable/jupyter/values.yaml]: Please change it to `maxExecutorCores`

File: stable/zeppelin/values.yaml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 spark:
   hostname: spark-master
   port: 7077
-  executorCores: 2
+  executorCores: 1
```
Comment (bivas) [inline stable/zeppelin/values.yaml:4]: Same as above

File: stable/shell/values.yaml [hunk 3]
```diff
@@ -7,7 +7,9 @@ image:
 spark:
   hostname: spark-master
   port: 7077
-  executorCores: 2
+  executorCores: 1
+  coresMax: 4
```
Comment (bivas) [inline stable/shell/values.yaml]: As above

**Review by bivas — CHANGES_REQUESTED** [review]

File: stable/jupyter/values.yaml (hunk 1 above)
Comment (gshatz) [reply stable/jupyter/values.yaml]: coresMax is not per executor but per application. I don't mind change it to "appMaxCores".
WDYT?

**Pull request merged**